apps/store/extensions/app/social-reviews/modules/reviews-utils.js

```diff
@@ -43,37 +46,71 @@ var ReviewUtils = {};
         paging.limit = paging.limit ? paging.limit : REVIEW_PAGE_LIMIT;
         return paging;
     };
-    var processUserReviews = function(socialSvc, user, reviews) {
+    var processUserReviews = function (socialSvc, user, reviews, target) {
         var review;
-        var formattedUsername = formatUsername(user);
+        var myReview;
+        var socialSvc = getSocialSvc();
+        var username = formatUsername(user);
+        myReview = JSON.parse(String(socialSvc.getUserComment(username, target)));
+        if (!isEmpty(myReview)) {
+            myReview.actor.id = cleanUsername(myReview.actor.id);
+        } else {
+            myReview = false;
+        }
         var usernameOnReview;
+        var myIndex;
```

Comment:
It is a good coding practice to put all declarations at the top of each script or function. Other than that it is good practice to initialise them.